app/components/dex-edit.jsx

```diff
@@ -59,11 +75,17 @@ export class DexEdit extends Component {
     onRequestClose(shouldReload);
   }
 
-  get showGenerationWarning () {
+  get showGameWarning () {
     const { dex } = this.props;
     const { game } = this.state;
 
-    return game === 'omega_ruby' && dex.game.id === 'sun';
+    if ((dex.game.game_family.id === 'sun_moon') && NATIONAL_ONLY_GAMES.indexOf(game) > -1) {
+      return true;
+    } else if ((dex.game.game_family.id === 'ultra_sun_ultra_moon') && game !== 'ultra_sun' && game !== 'ultra_moon') {
```

Comment:
blech. ill think of way to make this better

Reply:
ya it is confoosing lol.